ddtrace/profiling/recorder.py

```diff
@@ -34,10 +34,22 @@ class Recorder(object):
 
     events = attr.ib(init=False, repr=False, eq=False)
     _events_lock = attr.ib(init=False, repr=False, factory=nogevent.DoubleLock, eq=False)
-    _pid = attr.ib(init=False, repr=False, factory=os.getpid)
 
     def __attrs_post_init__(self):
+        # type: (...) -> None
         self._reset_events()
+        forksafe.register(self._after_fork)
+
+    def _after_fork(self):
+        # type: (...) -> None
+        # NOTE: do not try to push events if the current PID has changed
```

Comment:
This is implied now by the function being a post fork hook maybe can say this instead:
```suggestion
        # NOTE: do not try to push events if the process forked
```

(and update the rest of the comments below)